src/serac/infrastructure/profiling.hpp

```diff
@@ -71,4 +94,29 @@ void initializeCaliper(const std::string& options = "");
  */
 void terminateCaliper();
 
+/// detail namespace
+namespace detail {
+/**
+ * @fn void serac::profiling::setCaliperMetadata(const std::string & name, double data)
```

Comment:
The doxygen comment for the first overload should have `brief` and `param` tags - I think all the remaining overloads need *only* an `overload` tag without the function name - I think it usually gets picked up automatically?

Reply:
It sort of picks it up. But it doesn't pick up the arguments. Instead it just fills in the following which I think is suitable.
> This is an overloaded member function, provided for convenience. It differs from the above function only in what argument(s) it accepts.

Reply:
Yep, that's what I would expect